static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -195,20 +189,27 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
     [api, replayId, orgSlug, projectSlug]
   );
 
-  const {isLoading: isErrorsFetching, data: errors} = useReplayErrors({
-    replayId,
-  });
+  const fetchErrors = useCallback(
+    async errorIds => {
+      const conditions = new MutableSearch('');
+      errorIds.forEach((errorId, i) => {
+        if (i > 0) {
+          conditions.addOp('OR');
+        }
+        conditions.addFilterValues('id', [errorId]);
+      });
 
-  useEffect(() => {
-    if (!isErrorsFetching) {
-      setState(prevState => ({
-        ...prevState,
-        fetching: prevState.fetching || isErrorsFetching,
-        isErrorsFetching,
-        errors,
-      }));
-    }
-  }, [isErrorsFetching, errors]);
+      const response = await api.requestPromise(`/organizations/${orgSlug}/events/`, {
+        query: {
+          field: ['id', 'error.value', 'timestamp', 'error.type', 'issue.id'],
+          projects: [projectSlug],
+          query: conditions.formatString(),
+        },
+      });
```

Comment:
Nothing to do for this PR but one thing to note is that we will want to limit these queries with a specific date range. I think these will be empty once we get passed the default range (14d?).